mudlet-mapper.xml

```diff
@@ -8423,6 +8473,34 @@ if not alreadyRun then mmp.updateIsles() end</script>
                             <string>mmapper changed continent</string>
                         </eventHandlerList>
                     </Script>
+                    <Script isActive="yes" isFolder="no">
+                        <name>Shackle handling</name>
+                        <packageName></packageName>
+                        <script>local shackle = shackle or {density = false}
+--Wear shackle function
+
+function mmpWearShackle()
```

Comment:
why not properly namespacing the functions? They could be `mmp.wearShackle` and `mmp.unwearShackle`.

Reply:
Probably because I meant to actually do it that way but didn't for some reason. I'll get it edited correctly.